metrics/density/README.md

```diff
@@ -0,0 +1,10 @@
+# Density metrics tests
+
+This section includes metrics tests focused on measure the amount of memory used by
+a determined number of containers or a specific process that is part of the flow.
+This number of containers could be working under different configurations/conditions.
+
+- `docker_memory_usage.sh`: measures Proportional Set Size memory average about a
+   number of containers launched in idle mode, It uses a `sleep` for a configurable
+   period of time to allow any memory optimisations to 'settle' (e.g. KSM execution).
+   This test uses smem tool to get the memory used.
```

Comment:
Some wording changes. Also, have some indicator of what `sleep` is. What is `settle`?/does it need to be literal text?: 
measures the Proportional Set Size (PSS) memory average for a
number of containers launched in idle mode. This test uses the ________ `sleep` for a configurable
period of time to allow any memory optimizations to 'settle' (e.g. KSM execution).

Reply:
Hi @iphutch `sleep` is command that pauses a certain time the execution of the tests script, it is very common/popular to use it, and about `settle` word, it makes reference to KSM execution. KSM is a linux module that looks for 2 or more programs running in the same environment, then it compares their memory and if there are identical memory pages, KSM will merged in one, this process takes time and if someone would like to check KSM effects needs to wait KSM ans after take the metric sample(sttle).

Reply:
Great, maybe this will work:

`docker_memory_usage.sh`: measures the Proportional Set Size (PSS) memory average for a
number of containers launched in idle mode. This test uses the`sleep` command to allow any memory optimizations to 'settle' (e.g. KSM execution) for a configurable period of time.

Reply:
cool! sounds better :)